.changeset/tame-steaks-own.md

```diff
@@ -5,6 +5,6 @@
 "@fluid-experimental/tree2": minor
 ---
 
-Rename "Value" Multiplicity and FieldKind
+tree2: Rename "Value" Multiplicity and FieldKind
```

Comment:
It seems odd that we need to prefix these package specific changeset files with the package they are in, since the package is listed up on line 5.

Also I thought these files got deleted as part of generating the release notes? Is there a second step we run after this that removes them?

Reply:
> It seems odd that we need to prefix these package specific changeset files with the package they are in, since the package is listed up on line 5.

Agreed. It's something the tools could do, but don't. It's a little trickier for changes that affect multiple packages (e.g. I don't want to prefix each change with a comma-delimited list of ten packages), but I agree the tools could automate this.

> Also I thought these files got deleted as part of generating the release notes? Is there a second step we run after this that removes them?

As part of generating the per-package changelogs, the changesets will be deleted. This PR generating the release notes is a manual process that I envision at some point being run by PM at some point. On the release side we shouldn't have to do anything except generate the stuff. The content and everything else should be reviewed elsewhere by more than just us - but that full process isn't real yet.